docs/developer/decision-records/2023-05-25-template-repository/README.md

```diff
@@ -0,0 +1,90 @@
+
+## Decision
+
+Two new repositories will be created within the EDC GitHub organization:
+- `.github` repository for common and universal documents,
+- dedicated [template repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository) for mandatory (repo-specific) files.
+
+## Rationale
+
+Currently, generally valid documents like the `pr_etiquette.md` or `CONTRIBUTING.md`
+are located at the Connector repository, although they apply to all repositories in the scope of the
+EDC project. In addition, resources like style files or issue/pr templates are duplicated across
+repositories and, with this, are not up-to-date, as most of the changes to those documents are made
+in the Connector repo and are not transferred accordingly.
+
+The reasoning for moving these documents out to separate repositories is as follows:
+- simplify maintenance of files
+- facilitate the entrance for new community members
+- improve automation of GitHub processes
+- solidify creation process of new EDC repositories
+
+## Approach
+
+At the time of this writing, two repositories can be identified:
+- `.github`
+- `template-basic`
+
+We need to identify "common" and "repository-specific" documents. In addition, we need to test how the
+`.github` repo and the repositories' `.github` folders relate to each other.
+
+### Common documents
+
+Suggested structure for `.github`:
+- `.github/`:
+    - [ISSUE_TEMPLATE](https://github.com/eclipse-edc/Connector/tree/main/.github/ISSUE_TEMPLATE) and [PULL_REQUEST_TEMPLATE.md](https://github.com/eclipse-edc/Connector/blob/main/.github/PULL_REQUEST_TEMPLATE.md)
+    - generic workflows, e.g.
+        - [scan-pull-request.yaml](https://github.com/eclipse-edc/Connector/blob/main/.github/workflows/scan-pull-request.yaml),
+        - [first-interaction.yml](https://github.com/eclipse-edc/Connector/blob/main/.github/workflows/first-interaction.yml),
+        - [close-inactive-issues.yml](https://github.com/eclipse-edc/Connector/blob/main/.github/workflows/close-inactive-issues.yml)
+        - ...
+- `contributing/`:
+    - [CONTRIBUTING.md](https://github.com/eclipse-edc/Connector/blob/main/CONTRIBUTING.md)
+    - [contribution_categories.md](https://github.com/eclipse-edc/Connector/blob/main/contribution_categories.md)
+    - [known_friends.md](https://github.com/eclipse-edc/Connector/blob/main/known_friends.md)
+    - [pr_etiquette.md](https://github.com/eclipse-edc/Connector/blob/main/pr_etiquette.md)
+    - [styleguide.md](https://github.com/eclipse-edc/Connector/blob/main/styleguide.md)
+    - ...
+- `docs/`: same as for every repo
+    - `developer/`: generic documentation files from [docs/developer](https://github.com/eclipse-edc/Connector/tree/main/docs/developer)
+        - `decision-records/`: those that cover the whole EDC project from [docs/developer/decision-records](https://github.com/eclipse-edc/Connector/tree/main/docs/developer/decision-records)
```

Comment:
i guess we would need both D-Rs that affect the EDC org, and D-Rs pertaining specifically to each repo. E.g. D-R about the release process (org-wide) vs. the usage of a specific lib (only a particular component)

Reply:
added some lines in 62-67